spec/services/work_version_event_description_builder_spec.rb

```diff
@@ -3,422 +3,468 @@
 require 'rails_helper'
 
 RSpec.describe WorkVersionEventDescriptionBuilder do
-  subject(:result) { described_class.build(form) }
+  subject(:result) { described_class.build(form:, event_type:) }
 
```

Comment:
This file didn't change as much as the diffs suggest.  Basically I put in two context blocks, one for all of the existing tests (representing an update to an existing work) that made all of the existing tests become indented, and a new context block for new tests related to the creation of a new work.